.github/workflows/release.yml

```diff
@@ -0,0 +1,61 @@
+---
+name: release
+
+on:
+  release:
+    types: [released]
+
+env:
+  AWS_DEFAULT_REGION: us-east-2
+  BUILD_REGION: us-east-2
+  CURL_CACHE_DIR: $HOME/.cache/curl
+  DEPLOY_REGION_KMS_MAP: "us-east-1:alias/cool/ebs,
+                          us-east-2:alias/cool/ebs,
+                          us-west-1:alias/cool/ebs,
+                          us-west-2:alias/cool/ebs"
+  PACKER_VERSION: 1.4.4
+  TERRAFORM_VERSION: 0.12.12
```

Comment:
It sure would be cool if we could find a nifty way to DRY out some of these environment variables that are the same in each of our workflow YML files... 😉

Reply:
I agree that would be nice.  I have looked.  There is no facility to do that at this time.  I expect that will change in the not-too-distant-future™.

Reply:
Just thought about this a bit more... We could do it using secrets.  It solves one problem but creates a pretty ugly other other one:  cannot see what versions we're using.   I hope they'll add support for external env files or an import/include syntax.

Reply:
We could also create a repo that holds our org-wide variables.  At runtime we could check it out and source it.   This could be a custom action.  Let me noodle on this some, and we'll revisit.

Reply:
The environment variables are now SUPER DRY.  Check out: f2fd977 and https://github.com/cisagov/setup-env-github-action